src/components/BorrowerProfile/LendCta.vue

```diff
@@ -625,6 +633,12 @@ export default {
 					this.matchingHighlightExpShown = version === 'b';
 				}
 			}
+		},
+		getDropdownERL() {
+			if (this.activeCookie !== '') {
+				return getCookieDropdown(this.activeCookie, this.userBalance, this.unreservedAmount);
+			}
+			return '25';
```

Comment:
It looks like this default fallback could be incorrect in some cases, for example, if the unreserved amount is less than 25.